cmd/hz/generator/handler.go

```diff
@@ -44,6 +44,7 @@ type HttpMethod struct {
 	RefPackage         string // handler import dir
 	RefPackageAlias    string // handler import alias
 	ModelPackage       map[string]string
+	GenHandler         bool // Whether to generate a handler, when an idl interface corresponds to multiple http method
```

Comment:
If here is to emphasize the generated handler size, better use "one" instead of "a".

Reply:
done